frontend/src/scenes/notebooks/Nodes/NotebookNodePlaylist.tsx

```diff
@@ -117,32 +117,26 @@ export const Settings = ({
     attributes,
     updateAttributes,
 }: NotebookNodeAttributeProperties<NotebookNodePlaylistAttributes>): JSX.Element => {
-    const { filters } = attributes
-    const [showAdvancedFilters, setShowAdvancedFilters] = useState(false)
+    const { filters, simpleFilters } = attributes
     const defaultFilters = getDefaultFilters()
 
-    const hasAdvancedFilters = useMemo(() => {
-        const defaultFilters = getDefaultFilters()
-        return addedAdvancedFilters(filters, defaultFilters)
-    }, [filters])
-
     return (
         <ErrorBoundary>
             <SessionRecordingsFilters
-                filters={{ ...defaultFilters, ...filters }}
-                setFilters={(filters) => updateAttributes({ filters })}
+                advancedFilters={{ ...defaultFilters, ...filters }}
+                simpleFilters={simpleFilters ?? {}}
+                setAdvancedFilters={(filters) => updateAttributes({ filters })}
+                setSimpleFilters={(simpleFilters) => updateAttributes({ simpleFilters })}
                 showPropertyFilters
-                onReset={() => updateAttributes({ filters: undefined })}
-                hasAdvancedFilters={hasAdvancedFilters}
-                showAdvancedFilters={showAdvancedFilters}
-                setShowAdvancedFilters={setShowAdvancedFilters}
+                onReset={() => updateAttributes({ filters: undefined, simpleFilters: undefined })}
             />
         </ErrorBoundary>
     )
 }
 
 type NotebookNodePlaylistAttributes = {
     filters: RecordingFilters
+    simpleFilters?: RecordingFilters
```

Comment:
Adding a new key doesn't cause any issues. All existing filters will be converted to "advanced"